test-workloads/src/bin/guestvm.rs

```diff
@@ -128,11 +133,43 @@ extern "C" fn kernel_init(_hart_id: u64, shared_page_addr: u64) {
             panic!("Cert parsing error");
         }
         Ok(cert) => {
+            let mut tcb_info_extn = DiceTcbInfo::default();
+            cert.tbs_certificate.extensions.as_ref().map(|extensions| {
+                for extn in extensions.iter() {
+                    if extn.extn_id != TCG_DICE_TCB_INFO {
+                        continue;
+                    }
+
+                    tcb_info_extn = match DiceTcbInfo::from_der(extn.extn_value) {
```

Comment:
Along the same lines as the above, it would be a bit more concise to use `expect()` rather than matching against the `Result<>`.